loadtests/loadtest/tutorial.py

```diff
@@ -0,0 +1,308 @@
+        record_id = unicode(uuid.uuid4())
+        resp = self.session.put(
+            self.record_url(record_id, 'default', collection_id),
+            data=json.dumps({'data': task}),
+            auth=self.auth,
+            headers={'Content-Type': 'application/json',
+                     'If-None-Match': '*'})
+        self.incr_counter("status-%s" % resp.status_code)
+        self.assertEqual(resp.status_code, 201)
+        record = resp.json()
+        self.assertEqual(record['data']['description'], task['description'])
+        self.assertEqual(record['data']['status'], task['status'])
+        self.assertIn('write', record['permissions'])
+
+        # Fetch the collection list and see the tasks (save the etag)
+        resp = self.session.get(
+            self.collection_url('default', collection_id),
+            auth=self.auth,
+            headers={'Content-Type': 'application/json'})
+        self.incr_counter("status-%s" % resp.status_code)
+        self.assertEqual(resp.status_code, 200)
+        records = resp.json()['data']
+        self.assertEqual(len(records), 2)
+        etag = resp.headers['ETag']
+
+        # Fetch the collection from the Etag and see nothing new
+        url = self.collection_url('default', collection_id)
+        resp = self.session.get(url,
+                                params={'_since': etag.strip('"')},
+                                auth=self.auth,
+                                headers={'Content-Type': 'application/json'})
+        self.incr_counter("status-%s" % resp.status_code)
+        self.assertEqual(resp.status_code, 200)
+        records = resp.json()['data']
+        self.assertEqual(len(records), 0)
+
+        # Update a task
+        resp = self.session.patch(
+            self.record_url(record_id, 'default', collection_id),
+            data=json.dumps({'data': {'status': 'done'}}),
+            auth=self.auth,
+            headers={'Content-Type': 'application/json'})
+        self.incr_counter("status-%s" % resp.status_code)
+        self.assertEqual(resp.status_code, 200)
+        record = resp.json()
+        self.assertEqual(record['data']['status'], 'done')
+        self.assertIn('write', record['permissions'])
+
+        # Try an update with If-Match on the saved ETag and see it fails
+        resp = self.session.patch(
+            self.record_url(record_id, 'default', collection_id),
+            data=json.dumps({'data': {'status': 'done'}}),
+            auth=self.auth,
+            headers={'Content-Type': 'application/json',
+                     'If-Match': etag})
+        self.incr_counter("status-%s" % resp.status_code)
+        self.assertEqual(resp.status_code, 412)
+        self.assertEqual(resp.headers['ETag'],
+                         '"%s"' % record['data']['last_modified'])
+
+        # Get the list of records and update the ETag
+        resp = self.session.get(
+            '%s?_since=%s' % (self.collection_url('default', collection_id),
```

Comment:
Wouldn't we want to store collection_url and record_url in a variable ? (*it seems repeated a lot*)